docs/content/manual/virtual-machines/clone-vm-and-don't-select-start-after-creation.md

```diff
@@ -1,11 +1,27 @@
 ---
 title: Clone VM and don't select start after creation	
 ---
+
+## Case 1
 1. Clone VM from Virtual Machine list
 
+### Expected Results
+## Expected Results
+1. Machine should start if start VM after creation was checked
+1. Machine should match the origin machine
+1. in Config
+1. In YAML
+1. You should be able to connect to new VM via console
+
+## Case 2
+1. Clone VM with volume from Virtual Machine list
+
+### Expected Results
```

Comment:
Is this pre-req?

Reply:
I updated the step  to be more specific.